### DISCUSSION TITLE
macOS logs

### DISCUSSION THREAD

**Discussion by kuglee** [wojciech-kulik/xcodebuild.nvim#59] [General]
Thanks for building this plugin.

I can see it in the Readme that macOS logging isn't supported. What is the technical limitation of this? Why can't the logs be redirected to .nvim/xcodebuild/app_logs.log? The macOS app simply prints to stdout.

**Comment by wojciech-kulik** [comment]
macOS logs still need some investigation. Maybe, it’s possible. 

On iOS the logs from the simulator come from `xcrun simctl` command and on physical device they come from lldb debugger. However, for macOS there is no extra command to run the app, so you just spawn the process or do it through lldb, but as far as I remember lldb doesn’t forward logs in this this case.

 I will investigate it next week unless someone else does that in the meantime :)

**Reply by kuglee** [reply]
No hurry. Keep up the good work 🙂

**Reply by wojciech-kulik** [reply]
Resolved in #63 :) Let me know if it works!

**Comment by kuglee** [comment]
It's working. Thanks 🙏.
